experiments/chaos/openebs_target_network_loss/test.yml

```diff
@@ -168,99 +168,69 @@
 
             - name: Check for the jiva controller container status
               shell: >
-                kubectl get pods {{ controller_pod.stdout }} -n {{ operator_ns }} 
+                kubectl get pods {{ ctrl_pod.stdout }} -n {{ operator_ns }} 
```

Comment:
in line 134, the var name is specified as `controller_pod.stdout`. check the name once

Reply:
@gprasath  can you  check the line no 167. it's ctrl_pod only. Line no 134 is for iscsi provisioner jiva. this one is for csi jiva